recap/converters/avro.py

```diff
@@ -352,7 +353,7 @@ def _parse(
                 # Short circuit so we don't re-register aliases for nested types
                 return return_type
             case _:
-                raise ValueError(f"Unsupported Avro schema: {avro_schema}")
+                return UnknownType(**extra_attrs)
```

Comment:
I think we should do `UnknownType(**avro_schema)` rather than extra_attrs. It'd be more useful for debugging to see everything in the Avro schema.